docker/packager/binary/Dockerfile

```diff
@@ -52,44 +31,77 @@ RUN git clone --depth 1 https://github.com/tpoechtrager/cctools-port.git \
     && cd cctools-port/cctools \
     && ./configure --prefix=/cctools --with-libtapi=/cctools \
         --target=aarch64-apple-darwin \
-    && make install \
+    && make install -j$(nproc) \
     && cd ../.. \
     && rm -rf cctools-port
 
-# Download toolchain and SDK for Darwin
-RUN wget -nv https://github.com/phracker/MacOSX-SDKs/releases/download/11.3/MacOSX11.0.sdk.tar.xz
+# !!!!!!!!!!!
+# END COMPILE
+# !!!!!!!!!!!
+
+FROM clickhouse/test-util:$FROM_TAG
+ENV CC=clang-${LLVM_VERSION}
+ENV CXX=clang++-${LLVM_VERSION}
+COPY --from=cctools /cctools /cctools
```

Comment:
@Felixoid shouldn't this command placed into the end, so that steps from this `0` stage can be built in parallel with building cctools?

Reply:
hmm, it may help, yes. Good catch!